force-app/main/triggers/classes/HOT_ServiceAppointmentHandler.cls

```diff
@@ -773,6 +774,24 @@ public without sharing class HOT_ServiceAppointmentHandler extends MyTriggers {
         }
     }
 
+    // Used to fix a bug in FSL app, see TOLK-2300 for details.
+    private static void addErrorIfUnassignedInterpreterChangeStatus(
```

Comment:
Trenger vi denne når du har endret flowen? Kan kanskje ha en en stund for å se om det blir noen feil. Kunne kanskje lagt til logging så kan vi lettere feilsøke om det blir noe i prod.